saleor/checkout/utils.py

```diff
@@ -879,20 +883,22 @@ def create_order(cart, tracking_code, discounts, taxes):
         'tracking_client_id': tracking_code,
         'total': cart.get_total(discounts, taxes)})
 
-    order = Order.objects.create(**order_data)
-
+    order = Order.objects.create(**order_data, checkout_token=cart.token)
     _fill_order_with_cart_data(order, cart, discounts, taxes)
     return order
 
 
-def is_fully_paid(cart: Cart):
-    payments = cart.payments.filter(is_active=True)
-    total_paid = sum(
-        [p.total for p in payments])
-    return total_paid >= cart.get_total().gross.amount
+def is_fully_paid(cart: Cart, taxes, discounts):
+    """Check if checkout is fully paid."""
+    payments = [
+        payment for payment in cart.payments.all() if payment.is_active]
+    total_paid = sum([p.total for p in payments])
```

Comment:
Same as the previous one, also could use `aggregate` here, since payments aren't needed to be loaded in memory. Something along the lines of: `filter(is_active=True).aggregate(Sum('total'))['total__sum']`

Reply:
Same as above, is someone e.g. lists orders in API which might call this method we will have database queries duplication.